tests/test_cli.py

```diff
@@ -169,3 +169,22 @@ def test_config_passed_down_to_command_children(
             "params": {"time": 5},
         }
     }
+
+
+def test_invalid_stomp_config_for_listner(runner: CliRunner):
+    result = runner.invoke(main, ["controller", "listen"])
+    assert result.exit_code == 1
+
+
+def test_valid_stomp_config_for_listner(runner: CliRunner):
+    with patch("uvicorn.run", side_effect=None):
+        result = runner.invoke(
+            main,
+            [
+                "-c",
+                "tests/example_yaml/valid_stomp_config.yaml",
+                "controller",
+                "listen",
+            ],
+        )
+    assert result.exit_code == 1
```

Comment:
The `controller listen` command connects to a message bus and then prints messages to stdout. It uses the `input` builtin to wait for the user to press Enter to terminate the process

Therefore could add change the invoke to
```
        result = runner.invoke(
            main,
            [
                "-c",
                "tests/example_yaml/valid_stomp_config.yaml",
                "controller",
                "listen",
            ], input="\n"
        )
```
to tell the testing to immediately terminate the process.

This will then cause the `exit_code` to be 0 instead of 1 which makes sense here.